src/context/CaptureDetailContext.js

```diff
@@ -83,7 +83,7 @@ export function CaptureDetailProvider(props) {
     }
 
     return api.getSpeciesById(speciesId).then((species) => {
-      setState({ ...state, species });
+      setState({ ...state, species: species[0] });
```

Comment:
I'd suggest doing the extraction within `api.getSpeciesById()` so that all callers get the species object or `null` (if `species.length === 0`) without having to extract it.